app/scripts/controllers/npSpecializations.js

```diff
@@ -347,9 +362,13 @@
       result.name = name;
 
       // additional properties
-      // result.okrId = okrId || null;
-      // result.npQualificationId = npQualificationId || null;
-      // result.actuality = actuality || null;
+      result.okrId = !okrId ? null : okrId;
+      result.actuality = !actuality ? false : actuality;
+      result.protocolNumber = !protocolNumber ? null : okrId;
+      // temp solution before backend fix problem with null protocolDate property
```

Comment:
protocolDate property of `NpViewModel` (input parameter of API `NpController.PostNp` method ) has nullable type `DateTime?`, but when client send `null` appears the next error in `ApiDateTimeConverter`:
![2017-06-08 01 35 43](https://user-images.githubusercontent.com/15806486/26924655-713104c0-4c4f-11e7-9564-a397eb710d6d.jpg)
Therefore `ModelState.IsValid = false`:
![2017-06-08 01 36 07](https://user-images.githubusercontent.com/15806486/26924713-94d43e88-4c4f-11e7-9cd5-624697e93c63.jpg)
And then server send BadRequest to client with following message:
![2017-06-08 01 37 31](https://user-images.githubusercontent.com/15806486/26924816-ffc0084e-4c4f-11e7-8e58-64fa2f181443.jpg)